GameLogic/engine/StandardGame.cs

```diff
@@ -17,11 +17,13 @@ public class StandardGame : IGameLogic<StandardGame>
         public StandardGame(string fenNotation)
         {
             this.gameState = StandardFENSerializer.DeserializeFromFEN(fenNotation);
+            IsValid();
         }
 
         public StandardGame(StandardGameState gameState)
         {
             this.gameState = gameState;
+            IsValid();
```

Comment:
What is the purpose of this call in the current form?

IIUC, even if it is not valid, nothing will break/alert/etc.

Reply:
Fixed!